Sources/BraintreeDropIn/Public/BraintreeDropIn/BTDropInUICustomization.h

```diff
@@ -0,0 +1,98 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Braintree Drop-in color scheme
+typedef NS_ENUM(NSInteger, BTDropInColorScheme) {
+    /// Light color palette
+    BTDropInColorSchemeLight,
+    /// Dark color palette
+    BTDropInColorSchemeDark,
+    /// Light or dark color palette, depending on the user's light or dark mode system preference. Only available in iOS 13+.
+    BTDropInColorSchemeDynamic API_AVAILABLE(ios(13.0))
+};
+
+/**
+ Options for customizing Braintree Drop-in's user interface.
+ */
+@interface BTDropInUICustomization : NSObject
+
++ (instancetype)new __attribute__((unavailable("Please use initWithColorScheme:")));
+
+- (instancetype)init __attribute__((unavailable("Please use initWithColorScheme:")));
+
+/// Creates an instance of BTDropInUICustomization with the given color scheme.
+///
+/// @param colorScheme The color scheme used to determine the default colors used. These defaults may be overriden by setting additional properties on BTDropInUICustomization.
+///
+/// @return UI customization options for Braintree Drop-in
+- (instancetype)initWithColorScheme:(BTDropInColorScheme)colorScheme;
```

Comment:
My thought with this initializer is that default values will be set based on the color scheme when the `BTDropInUICustomization` is initialized. And then merchants can override specific properties as needed.

The way things currently work with `BTUIKAppearance`, you get different results based on whether you set the color scheme first and _then_ set other properties or vice versa. (Basically, whatever you do last gets priority, but it's a little weird that the order of operations matters.)